core/station/impl/src/services/system.rs

```diff
@@ -386,25 +446,71 @@ impl SystemService {
     pub async fn init_canister(&self, input: SystemInit) -> ServiceResult<()> {
         let mut system_info = SystemInfo::default();
 
-        if input.admins.is_empty() {
-            return Err(SystemError::NoAdminsSpecified)?;
-        }
-
-        if input.admins.len() > u16::MAX as usize {
-            return Err(SystemError::TooManyAdminsSpecified {
-                max: u16::MAX as usize,
-            })?;
+        match &input.initial_config {
+            InitialConfig::WithAllDefaults {
+                admin_quorum,
+                operator_quorum,
+                users,
+            } => {
+                // adds the default admin group
+                init_canister_sync_handlers::add_default_groups();
+                // registers the admins of the canister
+                init_canister_sync_handlers::set_initial_users(users.clone(), &[*ADMIN_GROUP_ID])?;
```

Comment:
shouldn't we also add the `OPERATOR_GROUP_ID`? because admins should be able to approve things with the operator policy no?